tests/test-theme-templates.php

```diff
@@ -262,7 +262,12 @@ public function test_localize_verse() {
 			<pre class="wp-block-verse">Here is some <strong>verse</strong> to localize</pre>
 		<!-- /wp:verse -->';
 		$new_template      = CBT_Theme_Templates::escape_text_in_template( $template );
-		$this->assertStringContainsString( "<?php esc_html_e('Here is some <strong>verse</strong> to localize', '');?>", $new_template->content );
+
+		$expected_output = '<!-- wp:verse -->
+			<pre class="wp-block-verse"><?php /* Translators: %s are html tags */ echo sprintf( esc_html__( \'Here is some %sverse%s to localize\', \'\' ), \'<strong>\', \'</strong>\' ); ?></pre>
+		<!-- /wp:verse -->';
+
+		$this->assertStringContainsString( $expected_output, $new_template->content );
 	}
```

Comment:
Let's add a test case for this `If the content itself is having a %s, it breaks the page because of the improper closings.`

Reply:
Done here: [4504111](https://github.com/WordPress/create-block-theme/pull/719/commits/4504111b0900b9d377984ab2794a297f94a9659e)